scripts/qa_crowbarsetup.sh

```diff
@@ -2656,7 +2656,7 @@ function custom_configuration()
             fi
 
             # assign neutron-network role to one of SLE12 nodes
-            if [ -n "$want_sles12" ] && [ -z "$hacloud" ] && [ -n "$want_neutronsles12" ] && iscloudver 5plus ; then
+            if [[ $want_sles12 && !$hacloud && $want_neutronsles12 && $(iscloudver 5plus) ]]; then
```

Comment:
`$(iscloudver 5plus)` is 0 if its true (bash semantic: exit code 0 => ok ; exit code 1 => error)
So its wrong to move it into the `[[ ]]`.

Negating it might just work but it seems even more wrong then.
So my recommendation is to write it as:
```
if [[ $want_sles12 && !$hacloud && $want_neutronsles12 ]] &&  iscloudver 5plus ; then
```